### PR TITLE
fix: added confirmation checkbox to transfer modal for ECDSA signature

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* Added confirmation checkbox to transfer modal for ECDSA signature

**Check list**
- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Changes**
image

- [ ] (ex: add user list)

### CODE REVIEW

File: src/components/balance/modals/ModalTransferAmount.vue [hunk 1]
```diff
@@ -219,7 +210,6 @@ export default defineComponent({
       const chain = chainInfo ? chainInfo.chain : '';
       return getProviderIndex(chain);
     });
-    const isH160 = computed(() => store.getters['general/isH160Formatted']);
```
Comment (hoonsubin) [inline src/components/balance/modals/ModalTransferAmount.vue:222]: can you tell me what is the difference between `const isH160 = computed(() => store.getters['general/isH160Formatted']);` and `const isEthWallet = computed(() => store.getters['general/isEthWallet']);`?

Comment (impelcrypto) [reply src/components/balance/modals/ModalTransferAmount.vue:222]: isH160: `true` whenever the user want to have an EVM transaction such as H160 to H160 transaction by Ethereum wallet (`false` for SS58 transaction such as dAppsStaking).

isEthWallet: `true` whenever the user connects to the portal by Ethereum wallet

**Review by hoonsubin — APPROVED** [review]
LGTM

**Review by bobo-k2 — APPROVED** [review]
LGTM

**Pull request merged**